JustSaying/JustSayingBus.cs

```diff
@@ -167,22 +167,21 @@ private IMessagePublisher GetActivePublisherForMessage(Message message)
             {
                 activeRegion = Config.GetActiveRegion();
             }
-            _log.LogInformation($"Active region has been evaluated to {activeRegion}");
+            _log.LogInformation("Active region has been evaluated to '{Region}'.", activeRegion);
```

Comment:
Given how often this gets logged and rarely changes, maybe we should either:
  1. Only log if it _changes_ after the first log. Or:
  1. Log it at trace.

Reply:
Probably true. A separate PR maybe?

Reply:
Only for option 1. If we just want to make it Moreno verbose might as well just change it here.